packages/walt-docs/package.json

```diff
@@ -7,6 +7,7 @@
     "start": "webpack-dev-server -d --hot --inline",
     "start-container": "webpack-dev-server -d --hot --inline --host=$(hostname -I | xargs)",
     "build": "webpack && cp index.html explorer.js explorer.js.map walt.png ../../docs/",
+    "build:prod": "cross-env NODE_ENV=production webpack && cp index.html explorer.js explorer.js.map walt.png ../../docs/",
```

Comment:
There isn't much of a point to keep both prod and dev build commands so let's just use `build` for this, drop the old command entirely.

Reply:
I had similar thoughts. Will do the changes now as you have suggested. Thanks!